quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/cli/command/ShowConfig.java

```diff
@@ -100,28 +97,6 @@ private static Map<String, Set<String>> getPropertiesByGroup() {
                 .filter(ShowConfig::filterByGroup)
                 .collect(Collectors.groupingBy(ShowConfig::groupProperties, Collectors.toSet()));
 
-        StreamSupport.stream(getPropertyNames().spliterator(), false)
-                .filter(new Predicate<String>() {
-                    @Override
-                    public boolean test(String s) {
-                        ConfigValue configValue = getConfigValue(s);
-
-                        if (configValue == null) {
-                            return false;
-                        }
-
-                        return PersistedConfigSource.NAME.equals(configValue.getConfigSourceName());
-                    }
-                })
-                .filter(ShowConfig::filterByGroup)
-                .collect(Collectors.groupingBy(ShowConfig::groupProperties, Collectors.toSet()))
-                .forEach(new BiConsumer<String, Set<String>>() {
-                    @Override
-                    public void accept(String group, Set<String> propertyNames) {
-                        properties.computeIfAbsent(group, name -> new HashSet<>()).addAll(propertyNames);
-                    }
-                });
-
```

Comment:
Just out of curiosity - what was the purpose of this before? It seems all property names were already added to the map, right? And we don't need to check if it was persisted, or not. Is it correct?

Reply:
That was my read - we're iterating over the same set of names either way. Adding filtering won't expand what is included.